3D/senior-csharp-test/Assets/Cache/ICacheStorage.cs

```diff
@@ -1,9 +1,6 @@
 ﻿public interface ICacheStorage
 {
-	void Delete( string id );
-	void Set( byte[] data, string id, string version = null );
+	void Add( string id, byte[] data, string version );
+	void Remove( string id );
 	byte[] Get( string id );
-	bool Has( string id );
```

Comment:
why did you make these private?

Reply:
To reduce the number of things to be tested (I thought we wanted to reduce the tests so that the candidate does not take too much time on this).

But I think i should in fact remove everything related to the version (removed from CacheEntry constructor, from Add() parameters and remove MatchesVersion )

Reply:
done removed everything related to version, for the cache entry and for the cache file as well